<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>partMainStateLcd</class>
 <widget class="QWidget" name="partMainStateLcd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>185</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>185</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>185</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="statusLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldGrowthPolicy::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="rowWrapPolicy">
      <enum>QFormLayout::RowWrapPolicy::DontWrapRows</enum>
     </property>
     <property name="horizontalSpacing">
      <number>14</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelStatus">
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtStatus">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {color: palette(text);}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="status" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="axesLayout" rowstretch="0,0,0,0" columnstretch="1,6,6" columnminimumwidth="20,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>3</number>
     </property>
     <item row="0" column="2">
      <widget class="QLabel" name="labelWork">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <property name="text">
        <string>Work</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelMachine">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <property name="text">
        <string>Machine</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="txtWX">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>42</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Patopian 1986</family>
         <pointsize>31</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 31pt &quot;Patopian 1986&quot;;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>123.456</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextInteractionFlag::NoTextInteraction</set>
       </property>
       <property name="smallDecimalPoint" stdset="0">
        <bool>false</bool>
       </property>
       <property name="digitCount" stdset="0">
        <number>6</number>
       </property>
       <property name="value" stdset="0">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="txtMY">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>42</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Patopian 1986</family>
         <pointsize>31</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 31pt &quot;Patopian 1986&quot;;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>123.456</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextInteractionFlag::NoTextInteraction</set>
       </property>
       <property name="smallDecimalPoint" stdset="0">
        <bool>false</bool>
       </property>
       <property name="digitCount" stdset="0">
        <number>6</number>
       </property>
       <property name="value" stdset="0">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="txtWY">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>42</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Patopian 1986</family>
         <pointsize>31</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 31pt &quot;Patopian 1986&quot;;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>123.456</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextInteractionFlag::NoTextInteraction</set>
       </property>
       <property name="smallDecimalPoint" stdset="0">
        <bool>false</bool>
       </property>
       <property name="digitCount" stdset="0">
        <number>6</number>
       </property>
       <property name="value" stdset="0">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="txtMZ">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>42</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Patopian 1986</family>
         <pointsize>31</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 31pt &quot;Patopian 1986&quot;;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>123.456</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextInteractionFlag::NoTextInteraction</set>
       </property>
       <property name="smallDecimalPoint" stdset="0">
        <bool>false</bool>
       </property>
       <property name="digitCount" stdset="0">
        <number>6</number>
       </property>
       <property name="value" stdset="0">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="txtWZ">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>42</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Patopian 1986</family>
         <pointsize>31</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 31pt &quot;Patopian 1986&quot;;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>123.456</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextInteractionFlag::NoTextInteraction</set>
       </property>
       <property name="smallDecimalPoint" stdset="0">
        <bool>false</bool>
       </property>
       <property name="digitCount" stdset="0">
        <number>6</number>
       </property>
       <property name="value" stdset="0">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="txtMX">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>42</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Patopian 1986</family>
         <pointsize>31</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 31pt &quot;Patopian 1986&quot;;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Sunken</enum>
       </property>
       <property name="text">
        <string>123.456</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextInteractionFlag::NoTextInteraction</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
